pkg/sql/importer/import_stmt_test.go

```diff
@@ -1998,7 +1999,10 @@ func TestFailedImportGC(t *testing.T) {
 		SQLMemoryPoolSize:        256 << 20,
 		ExternalIODir:            baseDir,
 		Knobs: base.TestingKnobs{
-			GCJob: &sql.GCJobTestingKnobs{RunBeforeResume: func(_ jobspb.JobID) error { <-blockGC; return nil }},
+			GCJob: &sql.GCJobTestingKnobs{
+				RunBeforeResume:      func(_ jobspb.JobID) error { <-blockGC; return nil },
+				SkipWaitingForMVCCGC: true,
+			},
```

Comment:
why is this changed needed? It seems to me that this change will ensure the gc job uses delRange, whereas previously it does not.

Reply:
I was feeling lazy. I made it so if you set this knob then it implies that you should use the new protocol. I needed the new protocol so that the job status matched what the test expected. 

I could have injected the setting instead.

Reply:
similarly, I saw that we also changed a few other tests to ensure we use delRange in the gc job. Can you explain to me why? Thanks

Reply:
Can you clarify the question? Is the question "why do the tests expect the new protocol"?

Reply:
> so that the job status matched what the test expected. 
Then why did the test pass previously, when it didn't use the new protocol?

Reply:
Before this change, the default was to use the new protocol.

Reply:
So, is it because your change made the GC job respect the setting `storage.mvcc.range_tombstones.enabled`, and hence those test that previously used the new protocol will no longer use it. Thus, we need to make it use the new protocol by setting the knob here?

Reply:
Correct.